enclave/core/sgx/calls.c

```diff
@@ -421,6 +418,11 @@ static void _handle_ecall(
             arg_out = _handle_init_enclave(arg_in);
             break;
         }
+        case OE_ECALL_INIT_CONTEXT_SWITCHLESS:
+        {
+            arg_out = oe_handle_init_switchless(arg_in);
```

Comment:
Since switchless is initialized after enclave has been initialized, any switchless calls that are made by global constructors (constructors of global C++ objects or functions with __constructor__ attribute) within the enclave will not work.

Reply:
As agreed by Anand, we will track this in an issue and fix it later.

https://github.com/openenclave/openenclave/issues/2180